src/main/kotlin/app/cash/licensee/plugin.kt

```diff
@@ -43,7 +43,7 @@ class LicenseePlugin : Plugin<Project> {
     var foundCompatiblePlugin = false
     project.afterEvaluate {
       check(foundCompatiblePlugin) {
-        "'app.cash.licensee' requires compatible language/platform plugin to be applied"
+        "'app.cash.licensee' requires compatible language/platform plugin to be applied(project: ${project.name})"
```

Comment:
Let's use path since many projects can share a name.
```suggestion
        "'app.cash.licensee' requires compatible language/platform plugin to be applied (project: ${project.path})"
```